test/Core.Test/Services/OrganizationServiceTests.cs

```diff
@@ -320,6 +328,32 @@ await sutProvider.GetDependency<IMailService>().Received(1)
         await sutProvider.Sut.InviteUsersAsync(organization.Id, invitor.UserId, new (OrganizationUserInvite, string)[] { (invite, null) });
     }
 
+    [Theory]
+    [BitAutoData(OrganizationUserType.Admin)]
+    [BitAutoData(OrganizationUserType.Manager)]
+    [BitAutoData(OrganizationUserType.Owner)]
+    [BitAutoData(OrganizationUserType.User)]
+    public async Task InviteUser_TypesOtherThanCustom_UseCustomPermissionsFalse_Passes(OrganizationUserType inviteUserType, Organization organization, OrganizationUserInvite invite,
```

Comment:
I struggle with C# test names. How about `InviteUser_NonCustomType_Passes_WhenUseCustomPermissionsIsFalse`?

I won't hold this up over test naming, but I recommend thinking about how to make the test names here a bit more human readable.

Reply:
Andreas is writing some draft test guidelines here, they're not official and you don't have to follow them, but it might give you some ideas. https://bitwarden.atlassian.net/wiki/spaces/DEV/pages/312737793/WIP+Unit+test+conventions

Reply:
I've renamed the tests to be easier to understand whilst keeping the `Method_Condition_Result` naming convention. For example: `InviteUser_WithNonCustomType_WhenUseCustomPermissionsIsFalse_Passes`.

Reply:
Looks good, thanks!